packages/medusa/src/services/claim.ts

```diff
@@ -204,6 +209,102 @@ export default class ClaimService extends TransactionBaseService {
     )
   }
 
+  protected async validateCreateClaimInput(
+    data: CreateClaimInput
+  ): Promise<void> {
+    const lineItemServiceTx = this.lineItemService_.withTransaction(
+      this.manager_
+    )
+
+    const { type, claim_items, additional_items, refund_amount } = data
+
+    for (const item of claim_items) {
```

Comment:
Suggestion: lineItemService.list with id in ids and then loop for validation, that way there is only one query and then the validation